getting_hired/preparing_for_job_search/social_networking.md

```diff
@@ -0,0 +1,81 @@
+
+Your best strategy is to start conversations with a genuine interest in the person and connect on a mutual interest. For example, you could review someone’s profile on LinkedIn and find something in common, or begin a conversation in-person with questions related to what the person does, how they got to where they are, and what challenges they face in their daily work. Make it about them, not you. Show that you are interested in them as a person, not that you want them to give you something. Actively listen to what they say and ask follow-up questions and even offer to help with something if you are in a position to do so. Think of the interaction like dating, you don’t ask someone to be your partner in the first question (do you? lol), so the same applies here. If your initial conversation goes well, then you can follow up with a request to connect.
+
+If you find it hard to talk to strangers in person and you are on your way to a local meetup or conference, consider having some predetermined things to talk about. Come up with a few generic questions such as “Where do you work?”, “What tech stack do you use at work?”, “How did you get into coding?”. Also, don’t forget to prepare to talk about yourself. Think of a 30-second elevator pitch describing who you are, what you do, and where you want to be in 12 months. This will help put your mind at ease because you will have something to say when your brain doesn’t feel like thinking in the moment.
+
+If you’re still not sure how to get started virtually, find one or two people who seem to be doing a good job at connecting with others and study their behavior. How often do they post or comment on other posts? What sort of language do they use? How do they treat others that interact with them? Chances are, if you adapt some of these behaviors to become your own you will also have more success connecting with others. 
+
+### Maintain Your Network
+
+Your social network is a dynamic and organic entity. Much like a garden, it will take some maintenance and attention to keep it in shape. You can do this by asking and answering questions in online communities, sending messages to your connections, and attending in-person events or scheduling periodic coffee-chats with your closest connections. Ideally, this does not feel like a chore. Rather, it’s something you naturally do because you enjoy it and get value from the interactions.
+
+There is so much more to building a social network, but this lesson should give you a good start. Remember, it will take time, so start early and be genuine. Good luck!
+
+### Additional Resources
+
+This section contains helpful links to other content. It isn't required, so consider it supplemental.
+
+* How to use LinkedIn as a developer to get a job in tech - [video](https://www.youtube.com/watch?v=SG5Sb5WTV_g)
+* How to Network: a Guide for Remote Software Developers & Engineers - [blog post](https://arc.dev/developer-blog/how-to-network-as-remote-developer/)
```

Comment:
The link text should be the first bit of text for both. `[How to use LinkedIn...]()` and [How to Network...]()` instead of "video" and "blog post". Additionally, the items should provide a brief description of the resource/why it's useful.